tests/test_tarfs.py

```diff
@@ -96,15 +92,6 @@ def destroy_fs(self, fs):
         os.remove(fs._tar_file)
         del fs._tar_file
 
-    def assert_is_bzip(self):
```

Comment:
Why was this test deleted? (Looks like it needs to be renamed to `assert_is_gzip`)

Reply:
It is because it is not reachable:

<img width="529" alt="Screenshot 2019-08-06 at 18 45 46" src="https://user-images.githubusercontent.com/4280312/62563309-ad07c800-b87a-11e9-9ba2-79d460205a3b.png">

`fs` is not imported at all in `test_tarfs.py` so this function shall fail in your unit tests but nothing is failing at all. Hence it proves that it is **dead code**.

Reply:
I am wondering why your IDE is hiding this factor from your eyes in past?

Reply:
IMHO if a test isn't being run (for some reason - I've not looked into the problem) then the "fix" would be to change the unit-tests so the test _does_ get run, not simply delete the test?
ping @willmcgugan ?

> why your IDE is hiding this fact from your eyes in past

Not everyone uses an IDE :wink:  (and I'm sure different IDEs probably "show" different things)